data_steward/analytics/table_metrics/other_scripts/visit_date_disparity.py

```diff
@@ -0,0 +1,954 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# +
+# #!pip install --upgrade google-cloud-bigquery[pandas]
+# -
+
+from google.cloud import bigquery
+
+# %reload_ext google.cloud.bigquery
+
+client = bigquery.Client()
+
+# %load_ext google.cloud.bigquery
+
+# + endofcell="--"
+#######################################
+print('Setting everything up...')
+#######################################
+
+import warnings
+
+warnings.filterwarnings('ignore')
+import pandas_gbq
+import pandas as pd
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.lines as mlines
+from matplotlib.lines import Line2D
+
+from notebooks import parameters
+
+import matplotlib.ticker as ticker
+import matplotlib.cm as cm
+import matplotlib as mpl
+
+import matplotlib.pyplot as plt
+
+import os
+import sys
+from datetime import datetime
+from datetime import date
+from datetime import time
+from datetime import timedelta
+import time
+
+plt.style.use('ggplot')
+pd.options.display.max_rows = 999
+pd.options.display.max_columns = 999
+pd.options.display.max_colwidth = 999
+
+from IPython.display import HTML as html_print
+
+
+def cstr(s, color='black'):
+    return "<text style=color:{}>{}</text>".format(color, s)
+
+cwd = os.getcwd()
+cwd = str(cwd)
+print(cwd)
+
+print('done.')
+# -
+# --
+
+# +
+DATASET = parameters.Q2_2019
+
+print("Dataset to use: {DATASET}".format(DATASET = DATASET))
+
+# +
+dic = {
+    'src_hpo_id': [
+        "saou_uab_selma", "saou_uab_hunt", "saou_tul", "pitt_temple",
+        "saou_lsu", "trans_am_meyers", "trans_am_essentia", "saou_ummc",
+        "seec_miami", "seec_morehouse", "seec_emory", "uamc_banner", "pitt",
+        "nyc_cu", "ipmc_uic", "trans_am_spectrum", "tach_hfhs", "nec_bmc",
+        "cpmc_uci", "nec_phs", "nyc_cornell", "ipmc_nu", "nyc_hh",
```

Comment:
Same comment for this as the other PR, it would be better to retrieve this info from the table using get_hpo_info rather than manually adding them

Reply:
@nishanthpp93 I agree wholeheartedly. 

I, however, plan to address that in a future update as right now I am focused on generating the metrics for the NIH and this improvement is a bit lower on my priority list. 

This was the standard means for generating the site DF prior to me joining and it has unfortunately remained the status quo for almost a year.